thali/thalireplicationmanager.js

```diff
@@ -112,22 +117,32 @@ function networkChanged(status) {
   if (status.isAvailable && !this._isStarted) {
     this.start(this._deviceName, this._port, this._dbName);
   }
-}
+};
 
-/* synchronization */
+/**
+ * Synchronizes the peers from the peerAvailabiltyChanged event.
+ * @param {Object} peers Peers to sync which contain the peerIdentifier, peerAvailable and peerName.
+ */
+ThaliReplicationManager.prototype._syncPeers = function (peers) {
+  // Get a list of peers for later for checking if available
+  this._peers = peers.reduce(function (acc, peer) {
+    acc[peer.peerIdentifier] = peer; return acc;
+  }, {});
 
-function syncPeers(peers) {
   peers.forEach(function (peer) {
 
     var p = this._replications[peer.peerIdentifier];
```

Comment:
Could you please use a slightly more informative variable name than p? Like existingPeerReplication? I'm slow and keep getting confused between peer and p.